.changeset/beige-buses-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+"azure_role_assignments": major
+---
+
```

Comment:
can we elaborate about why the change is breaking and add a migration path for users?